internal/eval/eval.go

```diff
@@ -4723,98 +4723,98 @@ func parseJSONStructure(jsonData interface{}, nested bool) (resp []any) {
 // evalZADD adds all the specified members with the specified scores to the sorted set stored at key.
 // If a specified member is already a member of the sorted set, the score is updated and the element reinserted at the right position to ensure the correct ordering.
 // If key does not exist, a new sorted set with the specified members as sole members is created.
-func evalZADD(args []string, store *dstore.Store) []byte {
-	if len(args) < 3 || len(args)%2 == 0 {
-		return diceerrors.NewErrArity("ZADD")
-	}
+// func evalZADDOld(args []string, store *dstore.Store) []byte {
```

Comment:
Please remove the old eval function (commented code)

Reply:
Removed